examples/angular/patches/@angular+bazel+9.0.0-next.8.patch

```diff
@@ -33,3 +54,22 @@ index 9b88fbb..d2eb8aa 100755
          bundle_index_typings = ctx.actions.declare_file("%s.d.ts" % flat_module_out)
          declaration_files.append(bundle_index_typings)
          if is_legacy_ngc:
+@@ -603,7 +604,17 @@ def ng_module_impl(ctx, ts_compile_actions):
+     return providers
+ 
+ def _ng_module_impl(ctx):
+-    return ts_providers_dict_to_struct(ng_module_impl(ctx, compile_ts))
++    ts_providers = ng_module_impl(ctx, compile_ts)
++
++    # Add in new JS providers
```

Comment:
will you send the Angular PR also if you haven't already?

Reply:
already added to https://github.com/angular/angular/pull/32889